### DISCUSSION TITLE
关于链接精灵的建议

### DISCUSSION THREAD

**Discussion by cloudwalke** [windingwind/zotero-better-notes#1191] [Ideas]
### **目前链接精灵的使用场景：** 
我每一篇文章的条目笔记中都按照每个章节做了很多的学习整理，然后我用一个独立笔记把每篇文章条目笔记中的相同章节整理出来，方便学习，当我在独立笔记中阅读到不理解的内容时，想返回该章节所在条目笔记，因此需要有一个链接。
### **目前使用的不方便之处：**
一个是操作较为繁琐，另一个是一次操作只能单项指引，你只能知道该独立笔记中引用了哪些文献条目笔记中的章节，但是你阅读文献时，想知道这篇文章条目笔记中的这一块内容都被哪些独立笔记引用了，较为困难，而且链接太长，影响观感。因此，有一个建议。
### **建议和恳求**
增加一个彼此跳转的链接功能，比如在条目笔记中的光标处可以创建一个小图案，然后同时可以在其他笔记的任何位置创建一个相同的小图案，每次点击这个图案的时候，都会彼此跳转。这样会极大方便学习，个人建议，仅供参考，如能实现，将不胜感激！

**Comment by windingwind** [comment]
> 但是你阅读文献时，想知道这篇文章条目笔记中的这一块内容都被哪些独立笔记引用了，较为困难，而且链接太长，影响观感。因此，有一个建议。

<img width="360" alt="image" src="https://github.com/user-attachments/assets/cf0d1bdf-f1bd-4d8c-9857-a23d96322eb2">

入链即在其他笔记中引用了当前笔记的情况。

**Reply by cloudwalke** [reply]
但是这个功能貌似只有独立笔记中才有，而且只能看见整个笔记被哪些引用了，不能看出笔记中某一段落的内容被哪些独立笔记引用了。

而且使用链接精灵的时候，指向功能和提及功能只有一个能定位到光标，有一个只能是顶部或者是底部。就是现在的双向链接必须要操作两次才能完成，要是能一次就插入一个双向链接就好了。

**Reply by windingwind** [reply]
> 而且只能看见整个笔记被哪些引用了，不能看出笔记中某一段落的内容被哪些独立笔记引用了

如果笔记链接带有章节/行号，则会显示具体链接到的章节或行

> 指向功能和提及功能只有一个能定位到光标，有一个只能是顶部或者是底部

在当前笔记中插入链接时，始终插入在光标处；另一侧插入在可选择的各自章节首/尾。需要有不同标题层级方可形成章节结构

> 就是现在的双向链接必须要操作两次才能完成，要是能一次就插入一个双向链接就好了。

双向链接的本意不是两个笔记各自添加一个链接，而是每个链接可以同时在发起者和被链接者中显示，并可以互相转跳（如右侧栏所示）。没有必要两边都加链接。

**Comment by cloudwalke** [comment]
比如这条建议：https://github.com/windingwind/zotero-better-notes/discussions/884

**Comment by cloudwalke** [comment]
或者我应该应该如何编辑一段代码，将其放在独立笔记或者模板笔记中，使得其可以自动生成该指向该笔记的链接（类似链接精灵中的指向功能），然后我只需要粘贴到其他的独立笔记中就可以了呢

**Reply by windingwind** [reply]
编辑器右上角菜单-复制笔记链接

**Reply by cloudwalke** [reply]
![image](https://github.com/user-attachments/assets/2e8feea1-57c0-4217-809f-9cce81ffb661)
十分感谢您的回复和帮助，但是我在编辑器右上角菜单中没有找到复制笔记链接的功能? 再次感谢！

**Reply by windingwind** [reply]
复制节/复制行
